deisctl/config/config.go

```diff
@@ -106,13 +104,31 @@ func doConfigGet(client *etcdClient, root string, keys []string) ([]string, erro
 	return result, nil
 }
 
-// readSSHPrivateKey reads the key file and returns a base64 encoded string
-func readSSHPrivateKey(path string) (string, error) {
+// valueForPath returns the canonical value for a user-defined path and value
+func valueForPath(path string, v string) (string, error) {
 
-	bytes, err := ioutil.ReadFile(path)
-	if err != nil {
-		return "", err
+	// check if path is part of fileKeys
+	for _, p := range fileKeys {
+
+		if path == p {
+
+			// read value from filesystem
+			bytes, err := ioutil.ReadFile(utils.ResolvePath(v))
+			if err != nil {
+				return "", err
+			}
+
+			// see if we should return base64 encoded value
+			for _, pp := range b64Keys {
```

Comment:
So b64keys can only be used if they are also fileKeys, and only makes sense in that context? I guess that makes sense.